assets/prototype/application/hooks/useTimeZoneTime.ts

```diff
@@ -1,15 +1,15 @@
 import { zonedTimeToUtc, utcToZonedTime } from 'date-fns-tz';
-import useConfig from '../services/config/useConfig';
+import { configTypes, useConfig } from '../services/config/useConfig';
 
 const useTimeZoneTime = () => {
-	const { timezone } = useConfig();
+	const { timezone }: configTypes = useConfig();
 
 	const localTimeToUtc = (date: Date | string | number) => {
-		return zonedTimeToUtc(date, timezone);
+		return zonedTimeToUtc(date, timezone.name);
 	};
 
 	const utcToLocalTime = (isoDate: Date | string | number) => {
-		return utcToZonedTime(isoDate, timezone);
+		return utcToZonedTime(isoDate, timezone.name);
```

Comment:
This is amazing:
![image](https://user-images.githubusercontent.com/1477453/71264540-fdc11e80-234c-11ea-901d-66b53c79669b.png)

Reply:
not getting that same level of error detection in my IDE :(

Reply:
You can give a try to VS Code for some time, maybe you'll like it.
I see that it has `IntelliSense` support for PHP as well: https://code.visualstudio.com/docs/editor/intellisense